### PR TITLE
Add additional AspNet extensions

### PR DESCRIPTION
**Pull request by Abrynos**
Thank you in advance for considering the inclusion of this merge request.

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -2,9 +2,10 @@
 	<ItemGroup>
 		<PackageVersion Include="JetBrains.Annotations" Version="2021.3.0" />
 		<PackageVersion Include="IndexRange" Version="1.0.0" />
-		<PackageVersion Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="1.0.0" />
+		<PackageVersion Include="Microsoft.AspNetCore.Localization" Version="2.2.0" />
+		<PackageVersion Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.5" />
```
Comment (JustArchi) [inline Directory.Packages.props:6]: Link those dependencies to the smallest possible, for example `IServiceCollection` should be part of `Microsoft.Extensions.DependencyInjection.Abstractions` and we don't need to pull whole ASP.NET together with it.

Then to the **smallest possible version**, this is a library.

**Review by JustArchi — CHANGES_REQUESTED** [review]

File: Directory.Packages.props (hunk 1 above)
Comment (Abrynos) [reply Directory.Packages.props:6]: We still need it for our call to `IServiceCollection.AddMvcCore()`

Comment (JustArchi) [reply Directory.Packages.props]: Crap, I think we'll be taking this back to ASF then, I don't feel like it's justified pulling those two just for the sake of 2 helper functions.

I wish C# had something along suggested/recommended packages from Debian where we can just do something IF the package is provided :slightly_frowning_face:

**Comment by JustArchi** [comment]
The cost of adding those two is not worth the helper functions this PR provides, we're taking this back to ASF.

(Sorry for wasting your time thinking that this somehow won't require pulling whole ASP.NET core just for that)

**Pull request closed without merging**